README.md

```diff
@@ -28,19 +28,19 @@ entire rows or columns, all while keeping column type integrity.
 DataFrames can be constructed passing Series to the dataframe.New constructor
 function:
 
-```
+```go
 df := dataframe.New(
 	series.New([]string{"b", "a"}, series.String, "COL.1"),
 	series.New([]int{1, 2}, series.Int, "COL.2"),
 	series.New([]float64{3.0, 4.0}, series.Float, "COL.3"),
 )
 ```
 
-But as a general rule it is easier to load the data directly from
-other formats. The base loading function takes some records in the
+You also can load the data directly fromother formats. 
```

Comment:
Typo fromother -> from other:

You can *also* load the data directly from other formats.